src/components/ExamplesFromPkg.tsx

```diff
@@ -45,7 +45,7 @@ export const ExamplesFromPkg: Component<Props> = ({
 
   return (
     <Fragment>
-      <Heading level={2}>Examples</Heading>
+      <Heading level={2}>🤓 Examples</Heading>
```

Comment:
Not sure about this one - I don't yet see the link to examples, and I would love to avoid emojis representing specific human stereotypes.

Reply:
Good point: I was more trying to capture the excitement of the person, but I'll change it to 🔬 instead :)